### PR TITLE
[TE] frontend - aaronucsd/add comment, collapsible tables to dashboard

### PR DESCRIPTION
**Pull request by aaronucsd**
screen shot 2018-06-04 at 4 07 35 pm
screen shot 2018-06-04 at 4 07 39 pm

1. Added the alert's comment
2. Allow the metric tables to be collapsible by toggle button
3. clean up styles
4. updated to remove underscore's isEmpty() and use ember isBlank() instead
5. Add async/await to query-cache service

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss [hunk 1]
```diff
@@ -19,13 +19,23 @@
...
+    }
   }
 
   &__title {
-    margin: 30px 0 15px 0;
 
     span {
       margin-left: 5px;
+
+      .glyphicon {
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: Placeholder?

File: thirdeye/thirdeye-frontend/app/pods/home/template.hbs [hunk 2]
```diff
@@ -47,18 +47,23 @@
...
+      </div>
 
-          {{!-- {{#bs-accordion as |accordion|}}
-            {{#accordion.item
-              title=anomalyList.length
-              class="te-form__custom-label"
-            }} --}}
+      {{#bs-accordion as |accordion|}}
+        {{#each-in filteredAnomalyMapping as |metric humanizedEntityList|}}
+          {{!-- <h4 class="dashboard-container__title">Metric:
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/home/template.hbs]: Saving for later?

Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/home/template.hbs]: nah. Let me remove. Thanks!

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss (hunk 1 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: nope. Let me remove. Thanks!

**Pull request merged**